symbols.go

```diff
@@ -0,0 +1,1032 @@
+// Code generated by go generate; DO NOT EDIT.
```

Comment:
This file is not used.
Please remove it.

I think it's generate by the `//go:generate sh ...` in the `pkg/exchange/okex/convert.go` at line 18

Reply:
ok